hw/top_earlgrey/fpv/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -0,0 +1,148 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the master cfg hjson for FPV. It imports ALL individual FPV
+  // cfgs of the IPs and the full chip used in top_earlgrey. This enables to run
+  // them all as a regression in one shot.
+  name: top_earlgrey_batch_fpv
+
+  import_cfgs: [// common server configuration for results upload
+                "{proj_root}/hw/data/common_project_cfg.hjson"]
+
+  use_cfgs: [// TODO: implement some switch to only select "_fpv" testbenches
+             // TODO: if we default "_fpv" cov to be on, and the rest of the tbs cov off, need a
+             // command-line switch to disable cov.
+             {
+               name: prim_arbiter_ppc_fpv
+               fusesoc_core: lowrisc:fpv:prim_arbiter_ppc_fpv
+               import_cfgs: ["{proj_root}/hw/formal/data/common_fpv_cfg.hjson"]
+             }
+             {
+               name: prim_arbiter_tree_fpv
+               fusesoc_core: lowrisc:fpv:prim_arbiter_tree_fpv
+               import_cfgs: ["{proj_root}/hw/formal/data/common_fpv_cfg.hjson"]
```

Comment:
shouldn't these guys all have a `rel_path: "hw/ip/<ip name>/fpv/{tool}"` field such that this gets published correctly?

Reply:
Hi Michael, thanks for your question, we do not publish individual FPV module reports. We can only publish a summary.

Reply:
Ah ok that is why this is not present. Never mind then.